site/styles/main.scss

```diff
@@ -52,6 +52,9 @@ html {
 }
 body {
 	font-family: "Rawline", $font-family-base;
+	min-height: 100vh;
```

Comment:
How much testing has this had across browsers and breakpoints? I'm cautious as I remember being burned by these units in the early days of browser support. @brynyShift72